src/amo/components/AddonAdminLinks/index.js

```diff
@@ -48,10 +48,15 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
       return null;
     }
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;
+    const isNotTheme = addon.type !== ADDON_TYPE_STATIC_THEME;
```

Comment:
I think we do not need this variable. Use `!isTheme` instead.

Reply:
yees ;p clean code level hard ;p